### PR TITLE
[Elastic Agent] Document usage inside a container

### PR DESCRIPTION
**Pull request by ruflin**
DO NOT REVIEW THE CONTENT YET

### CODE REVIEW

**Comment by ruflin** [comment]
As we lack a bit of documentation around the container use case I started to work on docs related to it. The part I was not sure on where to put it. We have the kubernetes case described on a high level and also the variables of the container command. But I didn't find a place where it describes how agent works in a container, where the find the logs and all the options around it. Thoughts?

**Comment by bmorelli25** [comment]
I think for now it makes sense to put this content in the Elastic Agent bucket, next to "Run Elastic Agent on Kubernetes". Right now our goal is to add as much content to the Fleet User Guide as we can, without worrying too much about the overall structure. We'll address these structural problems prior to the GA release.

**Comment by bmorelli25** [comment]
Let me know if I can help in any way with this. This is an important missing part of the docs 🚀

**Comment by bmorelli25** [comment]
Hey @ruflin 👋 Anything I can do to help with this?

**Comment by ruflin** [comment]
Thanks for the ping @bmorelli25 on this one. It is a PR I wanted to have completed already for a few weeks. There are two issues I currently have: I want to test that all the commands work as expected and every time I derail in finding some other issues which needs fixing. Writing docs is a great way to find bugs :-)

The other part is around structuring it properly and cleaning it up. Here I would really appreciate your help. The two commands I have not tested yet is the docker fleet-server command and the compose file but I hope to get this done today. Any chance you could take it over afterwards and push it over the line?

**Comment by ruflin** [comment]
@bmorelli25 I finally managed to test all the commands and adjust them the way that these are working. Could you take it from here?

**Comment by bmorelli25** [comment]
> Writing docs is a great way to find bugs :-)

:) It sure is!

> The other part is around structuring it properly and cleaning it up. Here I would really appreciate your help. Could you take it from here?

Absolutely! Thanks for all of the content!

**Review by bmorelli25 — APPROVED** [review]
Pushed some changes. Thanks again! I'll merge when green.

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:98]: @michalpristas Would be great if you could fill this in later on after we got this merged.

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:134]: It seems the logs part now exists twice?

**Review by ruflin — COMMENTED** [review]
Thanks Brandon for taking this over. LGTM!

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc
Comment (bmorelli25) [reply docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:134]: oops!

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc
Comment (bmorelli25) [reply docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:87]: Opened https://github.com/elastic/observability-docs/issues/891 to track

**Pull request merged**
